CWE/events/italy.txt

```diff
@@ -54,460 +99,232 @@ country_event = {
+  picture = "de_gasperi"
   fire_only_once = yes
 
 	trigger = {
 		tag = ITA
-		year = 1946 NOT = { year = 1947 }
-		OR = { government = democracy government = hms_government }
+		year = 1947  
+		NOT = { year = 1950 }
+		
+		ruling_party_ideology = conservative
+		
+		government = democracy 
+		
+		has_country_flag = accepted_marshall_plan_aid
 	}
 
-	mean_time_to_happen = { months = 2 }
-
-	option = {
-		name = EVT_8007734_A
-		any_pop = { consciousness = 5 }
-		set_country_flag = moderate_ICP
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
-		ai_chance = { factor = 0.8 }
-	}
+	mean_time_to_happen = { months = 5  }
 
 	option = {
-		name = EVT_8007734_B
-		any_pop = { militancy = 1.0 }
-		set_country_flag = radical_ICP 
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
+		name = "So it must be."
+		political_reform = parties_only_nominated
+		any_pop = { 
+			ideology = { factor = 0.02 value = conservative } 
 		}
-		ai_chance = { factor = 0.2 }
+		any_pop = { 
+			    limit = { OR = { pop_majority_ideology = socialist pop_majority_ideology = communist pop_majority_ideology = communist_social } }
+				scaled_consciousness = 2
+				scaled_militancy = 3
+			}
+		ai_chance = { factor = 1 }
 	}
-	
+   
 }
 
-#Italian Popular front
+#1948 Constitution
 country_event = {
-  id = 8007735
-  title = EVT_8007735_NAME
-  desc = EVT_8007735_DESC
-  picture = "nwo2_italian_popular_front"
-  fire_only_once = yes
+	id = 8007770
+	title = "The 1948 Constitution"
+	desc = EVTDESC8007770
+	picture = "nwo2_austrian_state_treaty"
+  
+	fire_only_once = yes
 
 	trigger = {
 		tag = ITA
-		year = 1947 NOT = { year = 1949 }
-		has_country_flag = moderate_ICP
-		OR = { government = democracy government = hms_government }
+		year = 1948
+		NOT = { year = 1950 }
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		has_country_flag = ITA_1948_republic_flag
 	}
 
-	mean_time_to_happen = { months = 2 }
+	mean_time_to_happen = { months = 1 }
 
 	option = {
-		name = EVT_8007735_A
-		ai_chance = { factor = 0.8 }
-		any_pop = { consciousness = 5 }
+		name = "Adopt the new Constitution!"
+		prestige = 25
+		
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
+		scaled_consciousness = -2
+		scaled_militancy = -2
 		}
-		set_country_flag = very_moderate_ICP 
-	}
+      
+		clr_country_flag = presidential_system_republic
+		clr_country_flag = semi_presidential_system_republic
+		clr_country_flag = parliamentary_system_republic
+			
+		set_country_flag = parliamentary_system_republic
 
-	option = {
-		name = EVT_8007735_B
-		ai_chance = { factor = 0.2 }
-		any_pop = { militancy = 3 }
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-		set_country_flag = very_radical_ICP 
+		ai_chance = { factor = 1 }
```

Comment:
Please do not have any historical events with factor = 1. All historical event must have at least some percentage chance for an ahistorical option.